blockwork/build/caching.py

```diff
@@ -176,8 +199,25 @@ def get_package_hash(self, package: str):
             self.map_package(package)
         return self.hash_map[package]
 
+
+def get_byte_size(path: str | Path) -> int:
+    'Get the size of a file or directory in bytes'
```

Comment:
I'm slightly surprised single quotes have passed lint, I thought there was a ruff rule enforcing double quotes?

Reply:
There is... but pre-commit doesn't pick it up